docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,99 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/created`  | Fires when a price list is created. | 
+| `store/priceList/updated/` | Fires when a price list is updated. | 
+| `store/priceList/activated/` | Fires when a price list is activated. | 
+| `store/priceList/deactivated/` | Fires when a price list is deactivated. | 
+| `store/priceList/deleted/` | Fires when a price list is deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/pricelist/created`
```

Comment:
Does case of `priceList` matter here?

Reply:
`store/pricelist/created` does exist.
![Screenshot 2024-03-07 at 10 31 42 AM](https://github.com/bigcommerce/docs/assets/64287776/6191a475-e4c7-4cf5-981c-974ea4d0f8df)